### PR TITLE
Change rpc example to working code

### PR DESCRIPTION
**Pull request by tobiaslins**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1180,7 +1180,7 @@ <h3 id="additional-apis"><a href="#additional-apis">Additional APIs</a></h3>
         with the exception that it only admits a single object
         as its parameters only:</p>
 
-        <pre><code>win.rpc.emit('hi there', {
```
Comment (CodeTheory) [inline index.html]: Isn't `win` just a variable for the `BrowserWindow`? https://github.com/zeit/hyper/blob/master/app/index.js#L128

**Comment by tobiaslins** [comment]
@CodeTheory
Ok you are right, while developing hyper plugins, you will need to use window!

**Comment by CodeTheory** [comment]
@tobiaslins I'm sorry for the delay on reply of this, I've been really busy.

Are you saying this should still be merged in then? I think that would contradict to how Hyper is written then 🤔

**Comment by tobiaslins** [comment]
@CodeTheory 
As i know, hyper is using _win_ internally.
If you want to emit rpc within a hyper plugin - you will need to use window :)

**Comment by CodeTheory** [comment]
Thanks @tobiaslins!

**Comment by ekmartin** [comment]
There's no `window` global available in Hyper plugins. You need to use `onWindow` which is passed the window instance as its first argument. Maybe a better change would be to wrap the code in that, to show where the user can get the `window` object from? 

I.e.
```js
exports.onWindow = win => {
  win.rpc.emit('hi there', {
    some: 'payload',
    any: [
      'object',
      'here'
    ]
  });
};
```

**Comment by tobiaslins** [comment]
@ekmartin 
window is available globally in plugins.
You can try to open the inspector in hyper and just type window!

We also used it in the plugin [hyperlayout](https://github.com/timolins/hyperlayout/blob/master/index.js)

**Comment by ekmartin** [comment]
That's because you're calling the functions from the browser context though (i.e. the React app) - `onWindow` is called in the main process (where `window` wouldn't be available).
Both are good examples nonetheless though, so lets just leave it :)

**Pull request merged**